src/main/java/com/ita/if103java/ims/service/impl/ItemServiceImpl.java

```diff
@@ -181,23 +176,11 @@ private boolean isLowSpaceInWarehouse(ItemTransactionRequestDto itemTransaction,
         if (volume == 0) {
             return true;
         } else {
-           return volume * 100 / warehouseDao.findById(itemTransaction.getDestinationWarehouseId(), accountId).getCapacity() < Float.parseFloat(maxWarehouseLoad);
+            return volume * 100 / warehouseDao.findById(itemTransaction.getDestinationWarehouseId(), accountId).getCapacity() > Float.parseFloat(maxWarehouseLoad);
```

Comment:
Shouldn't you also remove inversion of this method call?
I mean `isLowSpaceInWarehouse()` instead of `!isLowSpaceInWarehouse()`

Please double check!